ci/install.sh

```diff
@@ -7,16 +7,16 @@
 set -o errexit \
     -o pipefail
 
-wget https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge-pypy3-$(uname)-$(uname -m).sh \
+wget https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-$(uname)-$(uname -m).sh \
```

Comment:
Not sure why we were installing `Miniforge-pypy3` instead of `Miniforge3` before, probably just a mistake in https://github.com/manubot/rootstock/pull/450. `Miniforge-pypy3` includes `pypy` in addition to standard `python`, but I don't think we ever use `pypy`, so could save a bit of time just by switching to `Miniforge3`. But resolving the environment with `conda` would still be slow.